src/components/ContactPolicy/ContactPolicy.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { cleanup, fireEvent, render, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { ContactPolicy } from './ContactPolicy';
+
+describe('ContactPolicy component', () => {
+  afterEach(cleanup);
+
+  const ContactPolicyComponent = () => (
+    <DopplerIntlProvider>
+      <ContactPolicy />
+    </DopplerIntlProvider>
+  );
+
+  it('should the inputs be disabled if the switch is not active', async () => {
+    // Arrange
+    const { container } = render(<ContactPolicyComponent />);
+
+    const switchButton = container.querySelector('input#contact-policy-switch');
+    const inputAmount = container.querySelector('input#contact-policy-input-amount');
+    const inputInterval = container.querySelector('input#contact-policy-input-interval');
+
+    // Assert
+    expect(switchButton).not.toBeChecked();
+    expect(inputAmount).toBeDisabled();
+    expect(inputInterval).toBeDisabled();
+  });
+
+  it("shouldn't the inputs be disabled if the switch is active", async () => {
+    // Arrange
```

Comment:
```suggestion
    // Act
```

Reply:
Some test could be added to verify that when the form is sent, the success message is displayed